saleor/graphql/shipping/types.py

```diff
@@ -99,30 +100,42 @@ class ShippingMethod(ChannelContextTypeWithMetadata, CountableDjangoObjectType):
         "saleor.graphql.product.types.products.Product",
         description="List of excluded products for the shipping method.",
     )
+    id = graphene.ID(required=True, description="Shipping method ID.")
+    name = graphene.String(required=True, description="Shipping method name.")
+    description = graphene.JSONString(description="Shipping method description (JSON).")
+    minimum_order_weight = graphene.Field(
+        Weight, description="Minimum order weight to use this shipping method."
+    )
+    maximum_order_weight = graphene.Field(
+        Weight, description="Maximum order weight to use this shipping method."
+    )
+    maximum_delivery_days = graphene.Int(
+        description="Maximum number of days for delivery."
+    )
+    minimum_delivery_days = graphene.Int(
+        description="Minimal number of days for delivery."
+    )
 
     class Meta:
         default_resolver = ChannelContextType.resolver_with_context
         description = (
-            "Shipping method are the methods you'll use to get customer's orders to "
-            "them. They are directly exposed to the customers."
+            "External shipping method are the methods you'll use to get customer's "
```

Comment:
Why `external`?  From the customer perspective/storefront perspective, it doesn't matter if this is `External` or `built-in`. It expects the list of shipping methods.

Reply:
Good catch. Previously I have used this type only for external methods.